### PR TITLE
Bug 1935174: bump RHCOS boot image for LUKS fixes

### PR DESCRIPTION
**Pull request by miabbott**
This bumps the RHCOS boot images to address the LUKS problem
described in RHBZ#1934863

### CODE REVIEW

**Comment by miabbott** [comment]
Depends on https://github.com/openshift/installer/pull/4745 getting merged/verified

**Review by staebler — COMMENTED** [review]
/approve

**Comment by miabbott** [comment]
TL;DR - test failures do not look like they were introduced with this change

---

`e2e-metal-assisted` is pretty red lately; showing the following failure in most recent runs:

```
TypeError: __init__() got an unexpected keyword argument 'static_network_config'
make: *** [Makefile:295: _deploy_nodes] Error 1
make: *** [Makefile:299: deploy_nodes_with_install] Error 2
{"component":"entrypoint","error":"wrapped process failed: exit status 2","file":"prow/entrypoint/run.go:80","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2021-03-15T13:39:03Z"}
error: failed to execute wrapped command: exit status 2
```

Probably needs a new BZ.

`e2e-kubevirt` looks to be permafailing as of late.

`e2e-aws` is failing on `[sig-auth][Feature:SCC][Early] should not have pod creation failures during install [Suite:openshift/conformance/parallel]`

Sippy lead me to https://bugzilla.redhat.com/show_bug.cgi?id=1913069

`e2e-metal-ipi-ovn-dualstack` failing on `[sig-api-machinery] CustomResourcePublishOpenAPI [Privileged:ClusterAdmin] works for CRD preserving unknown fields in an embedded object [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]`

Seems like it has been present in various other CI jobs over the last 7 days; will file a BZ for it.

**Comment by miabbott** [comment]
> expected dependent Bugzilla bug 1934863 to target a release in 4.8.0, but it targets "4.7.z" instead

This doesn't make sense to me; the BZ is for a fix to part of RHCOS in 4.7, it should target 4.7.z

The same fix landed in RHCOS 4.8 as part of https://bugzilla.redhat.com/show_bug.cgi?id=1934174

**Comment by jlebon** [comment]
I've verified that the x86_64 RHCOS version here fixes https://bugzilla.redhat.com/show_bug.cgi?id=1935174.

/lgtm

**Comment by miabbott** [comment]
`e2e-openstack-ipi` appears to be permafailing well before this PR was introduced

`e2e-metal-assisted` also permafailing before this PR was introduced.  Filed https://bugzilla.redhat.com/show_bug.cgi?id=1939560

`e2e-kubevirt` also permafailing before this PR was introduced.

**Comment by staebler** [comment]
> `e2e-openstack-ipi` appears to be permafailing well before this PR was introduced
> 
> `e2e-metal-assisted` also permafailing before this PR was introduced. Filed https://bugzilla.redhat.com/show_bug.cgi?id=1939560
> 
> `e2e-kubevirt` also permafailing before this PR was introduced.

Yes, those should not be required jobs. They were introduced as required by a release bug.

/override ci/prow/e2e-openstack-ipi 
/override ci/prow/e2e-metal-assisted
/override ci/prow/e2e-kubevirt

**Comment by jlebon** [comment]
/hold

Discussing with @mike-nguyen, who was able to also verify the "rootfs too small" bug is fixed when testing RHCOS directly, but not when using the installer directly.

**Comment by lucab** [comment]
@miabbott @jlebon @mike-nguyen while further validation for the original request is ongoing, can you please refresh the artifacts to something at least as recent as RHCOS 47.83.202103181343? Full details in https://bugzilla.redhat.com/show_bug.cgi?id=1935174#c4.

**Comment by miabbott** [comment]
Oops, I forgot I had this open.  I started another, more up-to-date PR over at #4778
